docs/manual/index-checker.tex

```diff
@@ -75,6 +75,9 @@ \section{Index Checker structure and annotations\label{index-annotations}}
 \item
   whether an integer came from calling the JDK's binary search routine on
   an array (Section~\ref{index-searchindex})
+\item
+  whether an integer came from calling the JDK's \<String.indexOf> or
```

Comment:
This does not seem like a distinction as important as the others; it seems to just combine lower and upper bounds, so I would not list it separately.